### PR TITLE
ecg computation finalized

### PR DESCRIPTION
**Pull request by aungkonazim**

### CODE REVIEW

**Comment by twhnat** [comment]
Per our conversation, please let me know when the test files are corrected and I will merge these test cases into the master

File: cerebralcortex/data_processor/test/test_ecg.py [hunk 1]
```diff
@@ -23,25 +23,14 @@
...
-        pass
-
-    def test_filter_bad_ecg(self):
-        # TODO: Complete this test
-        pass
-
-    def test_compute_rr_intervals(self):
-        # TODO: Complete this test
-        pass
+        self._ecg_sample_array = np.genfromtxt(os.getcwd()+'/res/'+'sample_array_ecg.csv',delimiter=',')
```
Comment (twhnat) [inline cerebralcortex/data_processor/test/test_ecg.py]: You need to utilize this structure for pathing test files:

`os.path.join(os.path.dirname(__file__), 'res/ecg.csv.gz')`

**Comment by aungkonazim** [comment]
Hi Dr. Hnat,
I have taken some liberty to modify the ecg functions and uploaded the new testing files. Please take a glance and tell me what to change.
Thanks.

**Review by twhnat — CHANGES_REQUESTED** [review]
Need to fix TravisCI

**Review by twhnat — APPROVED** [review]

**Pull request merged**